mainnet/draft_genesis.json

```diff
@@ -0,0 +1,682 @@
+            "06-solomachine",
+            "07-tendermint"
+          ]
+        },
+        "create_localhost": false,
+        "next_client_sequence": "0"
+      },
+      "connection_genesis": {
+        "connections": [],
+        "client_connection_paths": [],
+        "next_connection_sequence": "0",
+        "params": {
+          "max_expected_time_per_block": "30000000000"
+        }
+      },
+      "channel_genesis": {
+        "channels": [],
+        "acknowledgements": [],
+        "commitments": [],
+        "receipts": [],
+        "send_sequences": [],
+        "recv_sequences": [],
+        "ack_sequences": [],
+        "next_channel_sequence": "0",
+        "params": {
+          "upgrade_timeout": {
+            "height": {
+              "revision_number": "0",
+              "revision_height": "0"
+            },
+            "timestamp": "600000000000"
+          }
+        }
+      }
+    },
+    "interchainaccounts": {
+      "controller_genesis_state": {
+        "active_channels": [],
+        "interchain_accounts": [],
+        "ports": [],
+        "params": {
+          "controller_enabled": true
+        }
+      },
+      "host_genesis_state": {
+        "active_channels": [],
+        "interchain_accounts": [],
+        "port": "icahost",
+        "params": {
+          "host_enabled": true,
+          "allow_messages": [
+            "*"
+          ]
+        }
+      }
+    },
+    "leveragelp": {
+      "params": {
+        "leverage_max": "10.000000000000000000",
+        "max_open_positions": "9999999999999",
+        "pool_open_threshold": "0.200000000000000000",
+        "safety_factor": "1.100000000000000000",
+        "whitelisting_enabled": false,
+        "epoch_length": "9999999999999",
+        "fallback_enabled": false,
+        "number_per_block": "100"
+      },
+      "pool_list": [],
+      "position_list": [],
+      "address_whitelist": []
+    },
+    "masterchef": {
+      "params": {
+        "lp_incentives": null,
+        "reward_portion_for_lps": "0.600000000000000000",
+        "reward_portion_for_stakers": "0.250000000000000000",
+        "max_eden_reward_apr_lps": "0.500000000000000000",
+        "supported_reward_denoms": [],
+        "protocol_revenue_address": "NEED THIS ADRESS"
+      },
+      "external_incentives": [],
+      "external_incentive_index": "0",
+      "pool_infos": [],
+      "pool_reward_infos": [],
+      "user_reward_infos": [],
+      "pool_rewards_accum": []
+    },
+    "oracle": {
+      "params": {
+        "band_channel_source": "channel-1",
+        "oracle_script_id": "37",
+        "multiplier": "18",
+        "ask_count": "4",
+        "min_count": "3",
+        "fee_limit": [
+          {
+            "denom": "uband",
+            "amount": "30"
+          }
+        ],
+        "prepare_gas": "600000",
+        "execute_gas": "600000",
+        "client_id": "band_price_id",
+        "band_epoch": "band_epoch",
+        "price_expiry_time": "86400",
```

Comment:
@avkr003 the price expiry time should be the shortest possible around 3-4 seconds aligned with blocktime

Reply:
Have to update it. Not final version. It's draft